go.mod

```diff
@@ -121,6 +121,7 @@ require (
 )
 
 replace (
+	golang.org/x/net => golang.org/x/net v0.0.0-20211209124913-491a49abca63 // CVE-2021-44716
```

Comment:
it's possible to achieve the same effect without replace directive
```
go get -v -d golang.org/x/net@491a49abca63
```

Reply:
Good point, done.